integration-tests/tests/src/tests/serialization.ts

```diff
@@ -39,26 +39,12 @@ import circularCollectionResult from "../structures/circular-collection-result.j
 import circularCollectionResultWithIds from "../structures/circular-collection-result-with-primary-ids.json";
 import { openRealmBeforeEach } from "../hooks";
 
-describe("JSON serialization (exposed properties)", () => {
-  it("JsonSerializationReplacer is exposed on the Realm constructor", () => {
-    expect(typeof Realm.JsonSerializationReplacer).equals("function");
-    expect(Realm.JsonSerializationReplacer.length).equals(2);
-  });
-});
-
 type TestSetup = {
   name: string;
   schema: (Realm.ObjectSchema | Realm.ObjectClass)[];
   testData: (realm: Realm) => unknown;
 };
 
-interface ICacheIdTestSetup {
```

Comment:
I am deleting these cached tests since we're not using a serializer... if we decide to add `flatted` tests here a format like this is probably useful.